components/infobox/wikis/rainbowsix/infobox_patch_custom.lua

```diff
@@ -58,4 +59,15 @@ function CustomPatch:addToLpdb(lpdbData)
 	return lpdbData
 end
 
+function CustomPatch:getChronologyData()
+	local data = {}
+	if _args.previous then
+		data.previous = _args.previous .. ' Patch|' .. _args.previous
+	end
+	if _args.next then
+		data.next = _args.next .. ' Patch|' .. _args.next
+	end
```

Comment:
Wiki code seems to use `|next link=`
https://liquipedia.net/rainbowsix/index.php?title=6.4.2_Patch&action=edit

Reply:
The links actually aren't really needed. having just ``|next=`` properly displays the patch and link. The only time the link would be necessary is if you wanted to have the next field display the patches name (ie Demon Veil) rather than the patch number. That could be solved with a redirect though

Reply:
```suggestion
	if _args.previous then
		data.previous = _args.previous .. '|' .. _args.previous
	end
	if _args.next then
		data.next = _args.next .. '|' .. _args.next
	end
```
This should be correct, no? The prev and next already contains `Patch` in the wikicode it looks like?

Reply:
the pages are `patchNumber_Patch` and in the previous/next params the input is only the patchNumber

Reply:
Ok, a few pages will be needed fixed/moved  (+ directs?) cc @FenrirWulf